### PR TITLE
Tweaks after beta drop

### PR DESCRIPTION
**Pull request by rdoxenham**
Pushing some readability changes and added some subtle context.

### CODE REVIEW

File: docs/03-explore.md [hunk 1]
```diff
@@ -1,47 +1,158 @@
...
-redirect it to a file instead of trying to look at it directly with the
-`logs` command.
+The Cluster Operator is the most important implementation within OpenShift, it's
+a self-hosted operator that is heavily responsible for installation, management,
+maintenance, and automated operations on the OpenShift cluster; it's essentially
+what turns Kubernetes into OpenShift - it enables all of the additional
+operators that we rely on (including the web console, the marketplace, image
+registry, and router service, etc) and configures the cluster as we have
+specified. We can view it's status by using the following command-
+
```
Comment (thoraxe) [inline docs/03-explore.md:91]: @acsulli can you look at the preceding paragraph for accuracy?

File: docs/04-scaling-cluster.md [hunk 2]
```diff
@@ -37,37 +74,115 @@ was running, which will temporarily make the console and other resources
...
-`machinesets` (they will look the same as in the web UI):
+You can alter the `Machine Set` count in several ways in the web UI console,
+but you can also perform the same operation via the CLI by using the `oc edit`
+command on the `machineset` in the `openshift-machine-api` project-
 
-    oc get machinesets -n openshift-machine-api
+~~~bash
+$ oc edit machineset -n openshift-machine-api
+(opens in vi)
+
```
Comment (thoraxe) [inline docs/04-scaling-cluster.md:107]: stylistically we usually split commands from their output into separate blocks.

File: docs/04-scaling-cluster.md [hunk 3]
```diff
@@ -37,37 +74,115 @@ was running, which will temporarily make the console and other resources
...
+NAME                                    DESIRED   CURRENT   READY     AVAILABLE   AGE
+beta-190305-1-79tf5-worker-us-east-1a   1         1         1         1           24h
+beta-190305-1-79tf5-worker-us-east-1b   1         1         1         1           24h
+beta-190305-1-79tf5-worker-us-east-1c   1         1         1         1           24h
+beta-190305-1-79tf5-worker-us-east-1d   0         0                               24h
+beta-190305-1-79tf5-worker-us-east-1e   0         0                               24h
+beta-190305-1-79tf5-worker-us-east-1f   0         0                               24h
+~~~
 
-### Define a `MachineAutoScaler`
```
Comment (thoraxe) [inline docs/04-scaling-cluster.md]: Why did you remove the header?

File: docs/04-scaling-cluster.md [hunk 4]
```diff
@@ -136,10 +252,23 @@ spec:
...
+To ensure you make no mistakes:
+
+~~~bash
+$ export CLUSTER_NAME=$(oc get machinesets -n openshift-machine-api | awk -F'-worker-' 'NR>1{print $1;exit;}')
+$ export REGION_NAME=us-east-1a
+
+$ sed -i s/\<aws-region-az\>/$REGION_NAME/g machine-autoscale-example.yaml
+$ sed -i s/\<clusterid\>/$CLUSTER_NAME/g machine-autoscale-example.yaml
+~~~
+
```
Comment (thoraxe) [inline docs/04-scaling-cluster.md:282]: OOOOOH thanks for this! I was meaning to figure it out.

File: docs/04-scaling-cluster.md [hunk 5]
```diff
@@ -213,43 +352,78 @@ Finally, allow the permissions, and then you will see the Grafana homepage.
...
-    ....
+~~~bash
+$ oc get machines -n openshift-machine-api
+NAME                                          INSTANCE              STATE     TYPE        REGION      ZONE         AGE
+beta-190305-1-79tf5-master-0                  i-080dea906d9750737   running   m4.xlarge   us-east-1   us-east-1a   26h
+beta-190305-1-79tf5-master-1                  i-0bf5ad242be0e2ea1   running   m4.xlarge   us-east-1   us-east-1b   26h
+beta-190305-1-79tf5-master-2                  i-00f13148743c13144   running   m4.xlarge   us-east-1   us-east-1c   26h
+beta-190305-1-79tf5-worker-us-east-1a-8dvwq   i-06ea8662cf76c7591   running   m4.large    us-east-1   us-east-1a   2m7s                    <--------
+beta-190305-1-79tf5-worker-us-east-1a-9pzvg   i-0bf01b89256e7f39f   running   m4.large    us-east-1   us-east-1a   2m7s                    <--------
+beta-190305-1-79tf5-worker-us-east-1a-vvddp   i-0e649089d42751521   running   m4.large    us-east-1   us-east-1a   2m7s                    <--------
```
Comment (thoraxe) [inline docs/04-scaling-cluster.md]: FYI because of the way GitHub wraps lines no one is likely to notice these arrows without reducing the whitespace between the time and the arrow.

**Review by thoraxe — CHANGES_REQUESTED** [review]
Overall looks pretty good. Requested some comments from Andrew and made some requests in the various comments. Small fixes and then we can merge.

Thanks for taking the time!

File: docs/03-explore.md (hunk 1 above)
Comment (acsulli) [reply docs/03-explore.md:91]: It's factually correct, but worded a bit strangely in my opinion.  Perhaps something like: 

_The cluster version operator is the core of what defines an OpenShift deployment.  The cluster version operator pod(s) contains the set of manifests which are used to deploy, updated, and/or manage the OpenShift services in the cluster.  This operator ensures that the other services, also deployed as operators, are at the version which matches the release definition and takes action to remedy discrepancies when necessary._

Also, are we no longer calling it the "cluster version operator"? "Cluster operator" refers to the operators deployed by the cluster version operator and can be viewed using the command `oc get clusteroperator`.

File: docs/03-explore.md [hunk 6]
```diff
@@ -60,19 +67,32 @@ ip-10-0-168-43.ec2.internal    Ready     master    23h       v1.12.4+5dc94f3fda
...
+The Cluster Operator is the most important implementation within OpenShift, it's
+a self-hosted operator that is heavily responsible for installation, management,
+maintenance, and automated operations on the OpenShift cluster; it's essentially
+what turns Kubernetes into OpenShift - it enables all of the additional
+operators that we rely on (including the web console, the marketplace, image
+registry, and router service, etc) and configures the cluster as we have
+specified. We can view it's status by using the following command-
 
 ~~~bash
 $ oc get deployments -n openshift-cluster-version
```
Comment (acsulli) [inline docs/03-explore.md:93]: This only indicates whether the pod has been deployed.  I think there are two more relevant commands.  The first is `oc get clusterversion`, which will output the current version of the OpenShift cluster and indicate a (very) high level status:

```console
[you@bastion]$ oc get clusterversion
NAME      VERSION     AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.0.0-0.7   True        False         28h     Cluster version is 4.0.0-0.7
```

The second command is `oc get clusterversion`, which provides status for the OpenShift cluster operators:

```console
[you@bastion]$ oc get clusteroperator
NAME                                  VERSION   AVAILABLE   PROGRESSING   FAILING   SINCE
cluster-autoscaler                              True        False         False     29h
cluster-storage-operator                        True        False         False     29h
console                                         True        False         False     28h
dns                                             True        False         False     29h
image-registry                                  True        False         False     29h
ingress                                         True        False         False     28h
kube-apiserver                                  True        False         False     29h
kube-controller-manager                         True        False         False     29h
kube-scheduler                                  True        False         False     29h
machine-api                                     True        False         False     29h
machine-config                                  True        False         False     17h
marketplace-operator                            True        False         False     29h
monitoring                                      True        False         False     80m
network                                         True        False         False     81m
node-tuning                                     True        False         False     28h
openshift-apiserver                             True        False         False     81m
openshift-authentication                        True        False         False     29h
openshift-cloud-credential-operator             True        False         False     29h
openshift-controller-manager                    True        False         False     29h
openshift-samples                               True        False         False     29h
operator-lifecycle-manager                      True        False         False     29h
```

File: docs/03-explore.md [hunk 7]
```diff
@@ -60,19 +67,32 @@ ip-10-0-168-43.ec2.internal    Ready     master    23h       v1.12.4+5dc94f3fda
...
 NAME                       DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
 cluster-version-operator   1         1         1            1           2h
 ~~~
 
-You can also `rsh` (remote shell access) into the running Operator and see the various manifests associated with the installed release of OpenShift:
+You can also `rsh` (remote shell access) into the running Operator and see the
+various manifests associated with the installed release of OpenShift:
 
 ~~~bash
 $ oc rsh -n openshift-cluster-version deployments/cluster-version-operator
```
Comment (acsulli) [inline docs/03-explore.md:237]: This is useful only in a very narrow set of use cases.  It might be worthwhile to replace, or expand, this section with the command `oc adm release info --commits`.  This will output the payload of the cluster version operator, providing the GitHub repoistory and specific commit at the point where the cluster version was "created".

This can be extremely useful for finding additional information about operators because it provides the location of that specific operator's documentation and code.

File: docs/03-explore.md (hunk 1 above)
Comment (rdoxenham) [reply docs/03-explore.md:91]: Thanks, I've used your wording instead. Please let me know if we need to change the "cluster version operator" wording.

File: docs/04-scaling-cluster.md (hunk 2 above)
Comment (rdoxenham) [reply docs/04-scaling-cluster.md:107]: OK, will do. Thx!

File: docs/04-scaling-cluster.md (hunk 3 above)
Comment (rdoxenham) [reply docs/04-scaling-cluster.md]: Added back in, not sure why this was removed.

File: docs/04-scaling-cluster.md (hunk 4 above)
Comment (rdoxenham) [reply docs/04-scaling-cluster.md:282]: No worries.

File: docs/03-explore.md (hunk 6 above)
Comment (rdoxenham) [reply docs/03-explore.md:93]: Thank you!

File: docs/03-explore.md (hunk 7 above)
Comment (rdoxenham) [reply docs/03-explore.md:237]: OK will add that in after I rebase. Thanks!

**Review by thoraxe — CHANGES_REQUESTED** [review]
Looks good pending your addition that you mentioned.

File: docs/03-explore.md (hunk 7 above)
Comment (rdoxenham) [reply docs/03-explore.md]: @acsulli - I don't think that command is fully implemented in the beta2 drop env that I have access to.

```
rdo$ oc adm release info --commits
Error: unknown flag: --commits


Available Commands:
  extract     Extract the contents of an update payload to disk
  new         Create a new OpenShift release

Use "oc adm <command> --help" for more information about a given command.

```

**Comment by rdoxenham** [comment]
> Looks good pending your addition that you mentioned.

All good. Done!

**Comment by thoraxe** [comment]
LGTM
